### PR TITLE
Corrections for correct work with ansible 2.8. Removed Debian 8 from support. Refactoring

### LINKED ISSUE #10: Error during running
ERROR! The requested handler 'restart mysql' was not found in either the main handlers list nor in the listening handlers list

ansible 2.8

### PR DESCRIPTION
**Pull request by y6vmeq**
## Description

* Fix #10 Corrections for correct work with ansible 2.8. Removed Debian 8 from support. 
* Refactoring

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

## Reviews

- [X] @tgadiev 
- [X] @pavelpikta 
 
## Checklist:

- [X] I have performed a self-review of my own code
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing tests pass with my changes

### CODE REVIEW

**Review by pavelpikta — APPROVED** [review]
LGTM

File: molecule/cloud-aws-direct-5.5/molecule.yml [hunk 1]
```diff
@@ -10,26 +10,26 @@ lint:
...
-#    assign_public_ip: False
-#    spot_price: 0.04
-#    security_group_name:
-#      - default
-#    wait_timeout: 1800
-#    ssh_user: centos
-#    groups:
-#      - rhel_family
+  - name: test-aws-centos7-mysql-5.5
+    image: ami-9887c6e7
```
Comment (tgadiev) [inline molecule/cloud-aws-direct-5.5/molecule.yml:14]: Add platform field

File: molecule/cloud-aws-direct-5.5/molecule.yml [hunk 2]
```diff
@@ -10,26 +10,26 @@ lint:
...
+    spot_price: 0.04
+    security_group_name:
+      - default
+    wait_timeout: 1800
+    ssh_user: centos
+    groups:
+      - rhel_family
 
   - name: test-aws-centos6-mysql-5.5
     image: ami-1585c46a
```
Comment (tgadiev) [inline molecule/cloud-aws-direct-5.5/molecule.yml:29]: Add platform field

File: molecule/cloud-aws-direct-5.6/molecule.yml [hunk 3]
```diff
@@ -15,7 +15,7 @@ platforms:
     instance_type: m5.large
```
Comment (tgadiev) [inline molecule/cloud-aws-direct-5.6/molecule.yml:16]: Add platform field

File: molecule/cloud-aws-direct-5.6/molecule.yml [hunk 4]
```diff
@@ -29,7 +29,7 @@ platforms:
     instance_type: m4.large
```
Comment (tgadiev) [inline molecule/cloud-aws-direct-5.6/molecule.yml:31]: Add platform field

File: molecule/cloud-aws-direct-5.6/molecule.yml [hunk 5]
```diff
@@ -43,21 +43,7 @@ platforms:
     instance_type: m5.large
```
Comment (tgadiev) [inline molecule/cloud-aws-direct-5.6/molecule.yml:46]: Add platform field

File: molecule/cloud-aws-direct-5.7/molecule.yml [hunk 6]
```diff
@@ -15,7 +15,7 @@ platforms:
     instance_type: m5.large
```
Comment (tgadiev) [inline molecule/cloud-aws-direct-5.7/molecule.yml:16]: Add platform field

File: molecule/cloud-aws-direct-5.7/molecule.yml [hunk 7]
```diff
@@ -29,7 +29,7 @@ platforms:
     instance_type: m4.large
```
Comment (tgadiev) [inline molecule/cloud-aws-direct-5.7/molecule.yml:31]: Add platform field

File: molecule/resources/provisioning/AWS/create.yml [hunk 8]
```diff
@@ -2,7 +2,7 @@
 - name: Create
```
Comment (tgadiev) [inline molecule/resources/provisioning/AWS/create.yml]: That file should be removed. We use external molecule drivers from its own repository.

File: molecule/resources/provisioning/AWS/destroy.yml [hunk 9]
```diff
@@ -2,7 +2,7 @@
 - name: Destroy
```
Comment (tgadiev) [inline molecule/resources/provisioning/AWS/destroy.yml]: File should be removed

**Review by tgadiev — CHANGES_REQUESTED** [review]
Please update tests with last ADK templates

**Review by tgadiev — APPROVED** [review]

**Pull request merged**